inference-engine/src/mkldnn_plugin/nodes/mkldnn_roi_pooling_node.cpp

```diff
@@ -114,15 +137,22 @@ struct jit_uni_roi_pooling_kernel_f32 : public jit_uni_roi_pooling_kernel, publi
     reg64_t reg_yoff = h_iter;
     reg64_t reg_xoff = r12;
 
+    Xbyak::Reg64 reg_load_table = r15;
+    Xbyak::Reg64 reg_load_store_mask = rcx;
+
     void roi_pool_max(int c_blocks) {
         Label h_loop_label;
         Label w_loop_label;
 
         mov(aux_reg_input, reg_input);
 
+        int src_c_off = jpp_.ih * jpp_.iw * jpp_.c_block * jpp_.src_data_size;
         for (int i = 0; i < c_blocks; i++) {
             Vmm vmm_max = get_acc_reg(i);
-            uni_vmovups(vmm_max, ptr[reg_input + i * jpp_.ih * jpp_.iw * jpp_.c_block * sizeof(float)]);
+
+            load_emitter->emit_code({static_cast<size_t>(reg_input.getIdx())}, {static_cast<size_t>(vmm_max.getIdx())},
+                                    std::make_shared<load_emitter_context>(jpp_.src_prc, Precision::FP32, step, false, "zero", i * src_c_off),
+                                    {}, {load_pool_gpr_idxs});
```

Comment:
Seems that the `load_pool_gpr_idxs` is a vector, so it is unnecessary to instantiate a new temporal vector from this one (just omit the braces).

Reply:
Done